.github/workflows/arm64-build-checks.yml

```diff
@@ -0,0 +1,57 @@
+name: arm64 build checks
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependency package
+      run: |
+        sudo sed -i -E 's|^deb ([^ ]+) (.*)$|deb [arch=amd64] \1 \2\ndeb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ \2|' /etc/apt/sources.list
+        sudo dpkg --add-architecture arm64
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+                crossbuild-essential-arm64 \
+                git \
+                cmake \
+                ca-certificates \
+                libpython-dev:arm64 \
+                libpython3-dev:arm64 \
+                libusb-1.0-0-dev:arm64 \
+                python-numpy \
+                python3-numpy \
+                libgstreamer1.0-dev:arm64 \
+                libgstreamer-plugins-base1.0-dev:arm64
+
+    - name: Fetch opencv_contrib
+      run: |
+        cd ..
+        git clone --depth 1 https://github.com/opencv/opencv_contrib.git
+
+    - name: Configure
+      run: |
+        mkdir build
+        cd build
+        cmake -DBUILD_SHARED_LIBS=ON \
+              -DBUILD_PERF_TESTS=ON \
+              -DBUILD_TESTS=ON \
+              -DWITH_OPENNI2=ON \
+              -DWITH_GDAL=ON \
+              -DBUILD_EXAMPLES=OFF \
+              -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 \
+              -DWITH_OPENCL=OFF \
+              -DBUILD_DOCS=OFF \
+              -DCMAKE_TOOLCHAIN_FILE=../platforms/linux/aarch64-gnu.toolchain.cmake \
+              -DWITH_CUDA=OFF \
+              -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules ../
+
+    - name: Build
+      run: |
+        cd build
+        cmake --build . --config release -- -j4
```

Comment:
Maybe just `make -j$(nproc --all)`?

Reply:
sure, will change it

Reply:
BTW,

Alternative is keeping `cmake --build . --config release` and using `Ninja` build tool (`-GNinja`).

- ninja doesn't don't mess error output in case of parallel builds (easy investigation of build failures)
- parallel mode is enabled automatically (`-j` option is not required)
- in general ninja is faster than make

---

Also it make sense to test "install" target instead of just building.
If GitHub Actions supports build artifacts (which can be downloaded later) then it would be nice to package install directory:
```
cmake -DCMAKE_INSTALL_PREFIX=$(pwd)/install ...
```
(see also [DESTDIR](https://cmake.org/cmake/help/v3.5/variable/CMAKE_INSTALL_PREFIX.html#variable:CMAKE_INSTALL_PREFIX))